x-pack/plugins/transform/public/app/sections/create_transform/components/advanced_runtime_mappings_settings/advanced_runtime_mappings_settings.tsx

```diff
@@ -0,0 +1,137 @@
+  } = props;
+
+  return (
+    <>
+      <EuiSpacer size="s" />
+
+      <EuiFormRow
+        fullWidth={true}
+        label={i18n.translate('xpack.transform.stepDefineForm.runtimeMappingsLabel', {
+          defaultMessage: 'Runtime mappings',
+        })}
+      >
+        <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+          <EuiFlexItem grow={true}>
+            {runtimeMappings !== undefined && Object.keys(runtimeMappings).length > 0 ? (
+              <FormattedMessage
+                id="xpack.transform.stepDefineForm.runtimeMappingsListLabel"
+                defaultMessage="{runtimeFields}"
+                values={{
+                  runtimeFields: Object.keys(runtimeMappings).join(','),
+                }}
+              />
+            ) : (
+              <FormattedMessage
+                id="xpack.transform.stepDefineForm.noRuntimeMappingsLabel"
+                defaultMessage="No runtime mapping"
+              />
+            )}
+
+            {isRuntimeMappingsEditorEnabled && (
+              <>
+                <EuiSpacer size="s" />
+                <AdvancedRuntimeMappingsEditor {...props} />
+              </>
+            )}
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false} style={{ width: advancedEditorsSidebarWidth }}>
+            <EuiFlexGroup gutterSize="xs" direction="column" justifyContent="spaceBetween">
+              <EuiFlexItem grow={false}>
+                <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+                  <EuiFlexItem grow={false}>
+                    <AdvancedRuntimeMappingsEditorSwitch {...props} />
+                  </EuiFlexItem>
+                  <EuiFlexItem grow={false}>
+                    <EuiCopy beforeMessage={COPY_TO_CLIPBOARD_RUNTIME_MAPPINGS} textToCopy={''}>
+                      {(copy: () => void) => (
+                        <EuiButtonIcon
+                          onClick={copy}
+                          iconType="copyClipboard"
+                          aria-label={COPY_TO_CLIPBOARD_RUNTIME_MAPPINGS}
+                        />
+                      )}
+                    </EuiCopy>
+                  </EuiFlexItem>
+                </EuiFlexGroup>
+              </EuiFlexItem>
+
+              {isRuntimeMappingsEditorEnabled && (
+                <EuiFlexItem style={{ width: advancedEditorsSidebarWidth }}>
+                  <EuiSpacer size="s" />
+                  <EuiText size="xs">
+                    {i18n.translate(
+                      'xpack.transform.stepDefineForm.advancedRuntimeMappingsEditorHelpText',
+                      {
+                        defaultMessage:
+                          'The advanced editor allows you to edit the runtime mappings of the transform configuration.',
+                      }
+                    )}
+                  </EuiText>
+                  <EuiSpacer size="s" />
+                  <EuiButton
+                    style={{ width: 'fit-content' }}
```

Comment:
https://caniuse.com/?search=fit-content have you tested Safari and Firefox?

Reply:
Removed here https://github.com/elastic/kibana/pull/90363/commits/76c9c799df5c5c7eb82011829fe57766336be7fd

Reply:
Seems like this is not supported in Firefox, but is supported on Chromium-based browsers and Safari. We can either remove 'fit-content' styling from this button and all other ones in Transform, or keep as is here.

Reply:
If it doesn't break the layout, let's keep it as is